.travis.yml

```diff
@@ -1,9 +1,10 @@
 language: python
 python:
-  - "3.5"
   - "3.6"
+  - "3.7"
 install:
-  - pip install --upgrade setuptools pip pytest pytest-cov coverage codecov
+  - pip install --upgrade vtk # fixes install of mayavi for Python-3.6, which is a dependency of boutdata
```

Comment:
Wait so this is a dependency only of our tests right? Does that mean it should be separated into some kind of tests_install section? We don't want users to install VTK if they don't need it? @ZedThree

Reply:
Mayavi is an optional dependency used in a couple of unmaintained modules. It can be safely ignored

Reply:
Mayavi's being pulled in when I pip-install `boutdata` for the test against `boutdata.collect`. Ignoring it probably requires an update to something (`setup.py`?) in https://github.com/boutproject/boutdata

PS If I understand right, `.travis.yml` is only being used to set up Travis for the CI tests, so this shouldn't affect users.